undefinedconfig.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
What does this file do, I see no references to it in the code.

Reply:
Whoops, that shouldn't be there.